packages/legend-query-builder/src/components/shared/BasicValueSpecificationEditor.tsx

```diff
@@ -705,6 +771,8 @@ const CollectionValueInstanceValueEditor = observer(
       setText(event.target.value);
     };
 
+    const placeholder = text === '' ? getPlaceHolder(expectedType) : undefined;
```

Comment:
i know it was here before but do we need the text === '' check. i think placeholder only shows when the input is empty regardless.